docs/architecture.adoc

```diff
@@ -75,6 +75,28 @@ DroneStateMapper::
 Maps the input from `RequestHandler` to expected `DroneState` for the `calculation layer`.
 ****
 
+****
+DronePhysics::
+This is a wrapper class, that contains the physical properties of the drone.
```

Comment:
```suggestion
This is a container class, that contains the physical properties of the drone.
```
A class that only stores data is called a container. A wrapper or adopter is used for interfaces.